lib/header.c

```diff
@@ -1863,13 +1862,7 @@ static rpmRC hdrblobVerifyRegion(rpmTagVal regionTag, int exact_size,
 	goto exit;
     }
 
-    /*
-     * Trailer offset is negative and has a special meaning.  Be sure to negate
-     * *after* the division, so the negation cannot overflow.  The parentheses
-     * around the division are required!
-     *
-     * Thankfully, the modulus operator works fine on negative numbers.
-     */
+    /* Negate after division to avoid overflow.  Watch out for precedence! */
```

Comment:
If you want to change this comment, put it into a separate commit (no need for separate PR though). Having it here makes it look like it's somehow related to this code change, when it isn't.